src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -227,15 +228,21 @@ static void Main()
     {
         var x = new (int, int)(1, 2);
         System.Console.WriteLine(x.ToString());
+
+        x = new (int, int)();
```

Comment:
Will add a test.
Also need a test to make sure the resulting type is a tuple type (not the underlying one)